modmail.py

```diff
@@ -257,8 +257,10 @@ def main(subreddit_name):
 			save_report_data(removing_mod, infraction, subreddit_name)
 
 			# Handle replying to the message with our private summary
-			reply = get_summary_text(user_infraction_db, user, subreddit_name, removing_mod)
-			send_reply(message, reply)
+			# ...but don'r reply if the action was taken by USL Bot
+			if removing_mod != "USLBot" :
```

Comment:
Please move this check to be after the send reply, and instead, conditionally archive the message automatically. 

So:

```
reply = get_summary_text
send_reply()

# Auto-archive USLBot actions
if removing_mod == 'USLBot':
   message.archive()
```

However, because mod mail is notoriously finicky and often throws 503's randomly, it might be good to wrap `message.archive()` in a method similar to `send_reply(message, reply)`